pwndbg/color/memory.py

```diff
@@ -16,14 +16,15 @@
 )
 
 
-def get(address, text=None) -> str:
+def get(address, text=None, prefix=None) -> str:
     """
     Returns a colorized string representing the provided address.
 
     Arguments:
         address(int): Address to look up
         text(str): Optional text to use in place of the address
               in the return value string.
+        prefix(str): Optional text to set at beginning in the return value string.
```

Comment:
Hmm shall we add the prefix here or elsewhere? (cc: @gsingh93 wdyt?)

Reply:
Yea, I feel like the prefix should be handled outside of this function.

Reply:
@feelfreelinux any chance u can revert that small change and handle it outside?